.github/workflows/main.yml

```
@@ -0,0 +1,57 @@
+  pull_request:
+    branches:
+      - main
+
+env:
+  REGISTRY: crretoxmas2024.azurecr.io
+  IMAGE_NAME: ithaka-backend
+
+jobs:
+  build-and-push:
+    runs-on: ubuntu-latest
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)